implementations/micrometer-registry-new-relic/src/main/java/io/micrometer/newrelic/NewRelicMeterRegistry.java

```diff
@@ -108,20 +108,20 @@ NewRelicClientProvider getClientProvider() {
         }
 
         /**
-         * Use the New Relic Java Agent-based client provider to publish metrics.
+         * Use the New Relic Insights Java Agent-based client provider to publish metrics.
          * @return builder
          * @since 1.4.0
          */
-        public Builder agentClientProvider() {
+        public Builder insightsAgentClientProvider() {
```

Comment:
Do we even need these methods anymore now that this is selected with the `NewRelicConfig` which is required to be passed to the Builder?

Reply:
I can actually remove them with a slight adjustment to where I put the constructor config check.  Will do.

Reply:
I left Builder and constructor support for providing a client provider.